src/sass/print.scss

```diff
@@ -270,6 +274,35 @@
     outline: 3px solid #000;
   }
 
+  /* Hiding options that were not selected and only showing the selection */
+  // .option-list {
+  //   .block label {
+  //     display: none;
+  //   }
+  //   .block label.checked {
+  //     display: block;
+  //   }
+  // }
```

Comment:
The above is commented out but this is how we can begin to hide unselected items from the print view 🤖